### PR TITLE
Heartbeats

### PR DESCRIPTION
**Pull request by dneise**
SH checks now on the webinterface for the last time the https://github.com/fact-project/shifthelper_heartbeat_monitor has updated its timestamp. 

If the SH see a too old timestamp (10minutes), it starts calling a **developer**, so we can make sure the "heartbeat monitor" is really running.

This PR is connected to a PR in the webinterface of course.
https://github.com/fact-project/shifthelper_webinterface/pull/29

### CODE REVIEW

**Comment by dneise** [comment]
should also fix #249

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='shifthelper',
-    version='1.1.4',
+    version='1.1.5',
```
Comment (MaxNoe) [inline setup.py]: I would suggest bumping the minor version (New feature release) → 1.2.0

File: shifthelper/notifiers.py [hunk 2]
```diff
@@ -97,14 +70,16 @@ def get_numbers_to_call(self, msg):
...
             numbers_to_call.append(self.phone_number_of_normal_shifter())
 
-            if self._get_oldest_call_age() >= self.time_before_fallback:
+            if self._get_oldest_message_age() >= self.time_before_fallback:
                 log.debug('Getting phone number of fallback shifter')
                 numbers_to_call.append(self.phone_number_of_fallback_shifter())
 
         return numbers_to_call
 
     def handle_message(self, msg):
```
Comment (MaxNoe) [inline shifthelper/notifiers.py]: This is much better, thanks

File: shifthelper/tools/__init__.py [hunk 3]
```diff
@@ -26,13 +26,19 @@
 db_engines = {}
 
 
-@retry(stop_max_delay=30000,  # 30 seconds max
-       wait_exponential_multiplier=100,  # wait 2^i * 100 ms, on the i-th retry
-       wait_exponential_max=1000,  # but wait 1 second per try maximum
-       )
 def get_alerts():
```
Comment (MaxNoe) [inline shifthelper/tools/__init__.py]: This means we never get called if the website is down. I think the developer should get called if the web service is not reachable for a longer time.

File: setup.py (hunk 1 above)
Comment (dneise) [reply setup.py]: okay sure... one sec

File: shifthelper/tools/__init__.py (hunk 3 above)
Comment (dneise) [reply shifthelper/tools/__init__.py]: oh

Comment (MaxNoe) [reply shifthelper/tools/__init__.py]: Or we say: If something happens, the shifter is called, realises the website is down and calls the expert. That woould minimize the number of calls the developer receives in the  night.

**Pull request merged**